README.md

```diff
@@ -22,6 +22,23 @@ applications that:
 
 To learn more about Habitat, please visit the [Habitat website](https://www.habitat.sh).
 
+## Installation
+
+There are a couple of different way to install Habitat.
+
+You can download it from the [Habitat downloads page](https://www.habitat.sh/docs/get-habitat/).
+
+Once you have downloaded it, follow the instructions on the page for your specific operating system.
+
+If you are running Linux, you can also use the Habitat install script.
```

Comment:
This script also currently works on Mac as well--it installs the raw `hab` binary to `/usr/local/bin/hab` so if you happen to be a homebrew user, you may even be able to drop the `sudo` call (it works either way though)